client/src/features/action/assets/Assets.tsx

```diff
@@ -78,8 +78,13 @@ function Assets(): JSX.Element {
     });
   };
 
+  const [donations, setDonations] = React.useState([]);
```

Comment:
i'm hoping this page will have a better design in the future to show just donations or just requests but for now let's implement getting both
so i'd say let's have 2 states, 2 fetches and each fetch will use a url like `/api/assets?type=request&search=${searchText}`
then on BE you can parse those query params via `@Query()` similar to `@Get()` in the transactions controller and select just a subset of the assets with each fetch

Reply:
Your tip to look at the `@Query()` in the transaction controller helped a lot, I think. I made another commit where I did my best to parse that out. Let me know what you think.